libs/langgraph/tests/test_pregel.py

```diff
@@ -590,7 +590,7 @@ def test_invoke_two_processes_in_out(mocker: MockerFixture) -> None:
 
 @pytest.mark.parametrize(
```

Comment:
should we move this to conftest ?

Reply:
ALL_CHECKPOINTERS_SYNC
ALL_CHECKPOINTERS_ASYNC

Reply:
yah sure, good idea